gatsby-config.js

```diff
@@ -156,14 +154,14 @@ module.exports = {
         // enablePartialUpdates: true,
         queries: require("./src/utilities/algoliaQueries"),
         // we skip the indexing completely on non-prod builds.
-        skipIndexing: process.env.CONTEXT != 'production',
```

Comment:
This is not working yet. This preview build was sent to the algolia index: https://app.netlify.com/sites/adapt-giving/deploys/611d80090d7e87000831b794#L101

Reply:
One other issue that could be a gotcha for developers is that when you use the netlify dev tool it will pull the environment vars from the environment you are working from. So if you are using the production environment as your remote, a developer may accidentally cause algolia indexing to run from their local host.

Reply:
🤔  Gotcha. I'll have to dig into this workflow a little more to get my head better wrapped around it.